tfe/resource_tfe_workspace.go

```diff
@@ -108,6 +108,12 @@ func resourceTFEWorkspace() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"remote_execution": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
```

Comment:
I tried to keep these parameters somewhat (not fully) alphabetically ordered. So in the spirit, this new parameter should probably be placed between `queue_all_runs` and `terraform_version`.